prime-router/src/main/kotlin/azure/Event.kt

```diff
@@ -103,20 +101,9 @@ abstract class Event(val eventAction: EventAction, val at: OffsetDateTime?) {
         fun parseQueueMessage(event: String): Event {
             // validate incoming queue message is in the expected format. This will error out with an
             //  IllegalStateException and message if it is not valid
-            val node: ObjectNode =
-                try {
-                    JacksonMapperUtilities.defaultMapper.readValue(event)
-                } catch (e: Exception) {
-                    return parseAndValidateOldQueueMessage(event)
-                }
-
-            val type = try {
-                node.get("type").asText()
-            } catch (e: Exception) {
-                return parseAndValidateOldQueueMessage(event)
-            }
+            val node: ObjectNode = JacksonMapperUtilities.defaultMapper.readValue(event)
```

Comment:
So I think you can can do this a little simpler now that you are only handling JSON messages

```
val message = JacksonMapperUtilities.defaultMapper.readValue<QueueMessage>(event)
            
            when (message) {
                is ReportEventQueueMessage ->
            }
...
```

Reply:
Queues were checked Friday and were empty. Cleaned up the code, good call!